build.gradle.kts

```diff
@@ -0,0 +1,69 @@
+import org.jetbrains.kotlin.gradle.tasks.KotlinCompile
+
+plugins {
+    id("org.springframework.boot") version "3.2.4"
+    id("io.spring.dependency-management") version "1.1.4"
+    id("org.graalvm.buildtools.native") version "0.9.28"
+    kotlin("jvm") version "1.9.23"
+    kotlin("plugin.spring") version "1.9.23"
+}
+
+group = "org.valensas"
+version = "0.0.1-SNAPSHOT"
+
+java {
+    sourceCompatibility = JavaVersion.VERSION_21
+}
+
+repositories {
+    mavenCentral()
+    if (project.hasProperty("GITLAB_REPO_URL")) {
+        maven {
+            name = "Gitlab"
+            url = uri(project.property("GITLAB_REPO_URL").toString())
+            credentials(HttpHeaderCredentials::class.java) {
+                name = project.findProperty("GITLAB_TOKEN_NAME")?.toString()
+                value = project.findProperty("GITLAB_TOKEN")?.toString()
+            }
+            authentication {
+                create("header", HttpHeaderAuthentication::class)
+            }
+        }
+    }
+}
+
+dependencies {
+    implementation("org.springframework.boot:spring-boot-starter-quartz:3.2.0")
+    implementation("org.springframework.boot:spring-boot-starter-jdbc")
+    implementation("org.reflections:reflections:0.10.2")
+
+    implementation("com.mchange:c3p0")
+
+    implementation("org.flywaydb:flyway-core")
+
+
+    runtimeOnly("org.postgresql:postgresql")
```

Comment:
Buna neden ihtiyacımız olmaması lazım. Hangi database kullanıldığı kütüphanenin umrunda olmamalı. Testler için gerekiyorsa `testRuntimeOnly` kullanılmalı ve Postgres yerine sqlite sanki daha kolay olur.

Reply:
test dependency lerinin hepsi silindi